src/Particles/babylon.particleSystem.ts

```diff
@@ -93,8 +83,7 @@
         public color2 = new Color4(1.0, 1.0, 1.0, 1.0);
         public colorDead = new Color4(0, 0, 0, 1.0);
         public textureMask = new Color4(1.0, 1.0, 1.0, 1.0);
-        public startDirectionFunction: (emitPower: number, worldMatrix: Matrix, directionToUpdate: Vector3, particle: Particle) => void;

```

Comment:
We need to keep them and use them if defined (it will overloads the emitter type in this case)
This is to avoid breaking changes

Reply:
Make sense, I should have been more cautious :)

Reply:
no worry :)